lease/lessor_test.go

```diff
@@ -140,13 +140,13 @@ func TestLessorRenew(t *testing.T) {
 	}
 
 	// manually change the ttl field
-	l.TTL = 10
+	l.SetTTL(10)
```

Comment:
Right, just fixed. `refresh` method calls are already protected with mutexes.

Just kept `TTL()` method for external packages.

PTAL. Thanks.